### PR TITLE
add Title and alt of images

### PR DESCRIPTION
**Pull request by BANCUNGUYE66**
-add page titles
-add image text alternatives
-Text headings- no issue found
-Color contrast- no issue found
-Resize no issue- found
-interaction - no issue found
-Moving content - no issue found
-Multimedia - no issue found
-the basic structure of the page - no issue found

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @BANCUNGUYE66, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - This is a pair-programming exercise and unfortunately, there are no commits from your partner. Please ask your partner to contribute to this exercise as the project required contributions from all the members.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

**Review by wayungi — COMMENTED** [review]
Hi @BANCUNGUYE66 , 
image

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ 

You have submitted a project that is invalid because of the following

- [ ] Delete all lines starting with `>`. These lines serve to guide you to fill in the document correctly and should be deleted afterward.
- [ ]  Add all the sections of the [README](https://github.com/microverseinc/readme-template) file. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by BANCUNGUYE66** [comment]
Hello @wayungi, please can you tell me what to do? i don't understand what to change. thank you!

File: index.html [hunk 1]
```diff
@@ -11,17 +11,17 @@
...
-    <title>Document</title>
+    <title>BANCUNGUYE's portfolio</title>
 </head>
 <body>
    <header>
       <nav>
           <div class="nav_bar">
              <div class="logo">
-                <a href="#"> Aima.b</a>
+                <a href="#"> <h1 class="logo-h1">Aima.b</h1></a>
```
Comment (leonardodiasb) [inline index.html]: - [x] The logo is correct in the navbar but the logo is not a `heading` tag. Please take a look [at this article](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) to understand better the hierarchy and the use of the `heading` tags and [this one](https://www.reliablesoft.net/h1-tag/) to understand better the use of the `h1` tag. The page should have one but no more than one `h1` tag. Kindly remove it for better SEO and semantics.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @BANCUNGUYE66, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [x] Kindly link the points you added following the tutorial in the link below and include the checks even if you did not find issues in your report to make it look professional. As the project required, your report should include all the topics you checked, i.e:
   - [Page titles](https://www.w3.org/WAI/test-evaluate/preliminary/#title)
   - [Image text alternatives** (alt text)](https://www.w3.org/WAI/test-evaluate/preliminary/#images)
   - [Text headings](https://www.w3.org/WAI/test-evaluate/preliminary/#headings)
   - [Color contrast](https://www.w3.org/WAI/test-evaluate/preliminary/#contrast)
   - [Resize](https://www.w3.org/WAI/test-evaluate/preliminary/#resize)
   - [Interaction](https://www.w3.org/WAI/test-evaluate/preliminary/#interaction) (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design)
   - [Moving content](https://www.w3.org/WAI/test-evaluate/preliminary/#moving)
   - [Multimedia](https://www.w3.org/WAI/test-evaluate/preliminary/#media)
   - [The basic structure of the page](https://www.w3.org/WAI/test-evaluate/preliminary/#structure)
 - If everything was ok, you should make a statement about that (e.g. "Page titles - no issues found").
 - If issues were found, you should [link to the corresponding line of code in your pull request changes](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -236,12 +236,12 @@ <h2>About<br> Myself </h2>
...
                      <div class="frameworks">Frameworks</div>
-                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt=""></div>
+                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt="My frameworks"></div>
                   </div>
                   <hr>
                   <div class="skills_bar">
                      <div class="skills">Skills</div>
-                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt=""></div>
+                     <div> <img src="images/arrow2/Normal Button/Tertiary/Icons/Union.png" alt="My skills"></div>
                   </div>
```
Comment (hasnatali1947) [inline index.html:245]: please consider wrapping all the **li tag** with **ul tage** to make it semantically correct 👍

File: index.html [hunk 3]
```diff
@@ -204,18 +204,18 @@ <h2>About<br> Myself </h2>
...
-                 <li> <img src="images/Vector_5.svg" alt=""> </li>
+                 <li> <img src="images/Vector_1.svg" alt="My twitter account"> </li>
+                 <li> <img src="images/Linkedin_icon.svg" alt="My linkedin account"> </li>
+                 <li> <img src="images/Vector_3.svg" alt="Medium"> </li>
+                 <li> <img src="images/Vector_4.svg" alt="My Github account"> </li>
+                 <li> <img src="images/Vector_5.svg" alt="Angellist"> </li>
                </ul>
                <div class="Project_button">
                   <button type="button">Get my resume</button>
                </div>
```
Comment (hasnatali1947) [inline index.html:215]: kindly you remove the button from **div** it will be semantically correct 👍

**Review by hasnatali1947 — COMMENTED** [review]
Change review ♻️ 
Please check my comments below and make as required changes.

Submit a re-review when you are done.

thanks 😄

**Pull request closed without merging**